File: GuardianMonsters/desktop/src/de/limbusdev/guardianmonsters/desktop/DesktopLauncher.java
package de.limbusdev.guardianmonsters.desktop;

import com.badlogic.gdx.backends.lwjgl.LwjglApplication;
import com.badlogic.gdx.backends.lwjgl.LwjglApplicationConfiguration;

import de.limbusdev.guardianmonsters.GuardianMonsters;
import de.limbusdev.guardianmonsters.utils.GuardianMonstersLML;

public class DesktopLauncher
{
	public static void main (String[] arg)
	{
		LwjglApplicationConfiguration config = new LwjglApplicationConfiguration();
		config.width = 1280;
		config.height = 720;
		config.title = "GuardianMonsters";
		//new LwjglApplication(new GuardianMonsters(), config);
		new LwjglApplication(new GuardianMonstersLML(), config);
	}
}

File: GuardianMonsters/desktop/src/de/limbusdev/guardianmonsters/desktop/DesktopLauncher.kt
package de.limbusdev.guardianmonsters.desktop

import com.badlogic.gdx.backends.lwjgl.LwjglApplication
import com.badlogic.gdx.backends.lwjgl.LwjglApplicationConfiguration

import de.limbusdev.guardianmonsters.GuardianMonsters

object DesktopLauncher {

    @JvmStatic
    fun main(arg: Array<String>) {

        val config = LwjglApplicationConfiguration()
        config.width = 1280
        config.height = 720
        config.title = "GuardianMonsters"
        LwjglApplication(GuardianMonsters(), config)
    }
}
